doc/cephfs/mds-config-ref.rst

```diff
@@ -97,7 +97,12 @@
 
 ``mds blacklist interval``
 
-:Description: The blacklist duration for failed MDSs in the OSD map.
+:Description: The blacklist duration for failed MDSs in the OSD map. Note,
+              this controls how long failed MDS daemons will stay in the
+              OSDMap blacklist. It has no effect on how long something is
+              blacklisted when the administrator blacklists it manually. For
+              example, ``ceph osd blacklist add`` will still use the default
+              blacklist time.)
```

Comment:
Seems there is an extra trailing `)` here?

Reply:
whoops, yes.

Reply:
Fixed in https://github.com/ceph/ceph/pull/18228